src/openzaak/components/besluiten/migrations/0014_besluitinformatieobject__objectinformatieobject_url.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.4 on 2019-12-24 11:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("besluiten", "0013_besluit__zaakbesluit_url"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="besluitinformatieobject",
+            name="_objectinformatieobject_url",
+            field=models.URLField(
+                blank=True,
+                help_text="URL of related IbjectInformatieObject object in the other API",
```

Comment:
```suggestion
                help_text="URL of related ObjectInformatieObject object in the other API",
```

This typo doesn't appear in models.py though

Reply:
Thanks! I probably fixed it in the models.py after `makemigrations` was used